tasty/graphs.py

```diff
@@ -144,3 +146,81 @@ def get_namespaced_term(ontology: Graph, term: str) -> Union[URIRef, bool]:
         ns = potential_namespaces[0]
         return ns[term]
     return False
+
+
+def print_as_hayson(graph: Graph):
+    """
+    Return the Hayson encoding of an rdf graph
+    :param graph: [rdflib.Graph]
+    :return:
+    """
```

Comment:
1. Thinking about our issues around naming, should we rename this something else to be more descriptive? Some ideas...

   - `graph_to_hayson_string`
   - `rdf_to_json_string`
   - ...

2. Add function annotation `-> str:` 
3. Add return `[str]`

Reply:
`rdf_to_json_string` is technically correct but might have unforeseen return types in usage.  Hayson is like JSON but like a specific encoding.  My vote would be for the first one `graph_to_hayson_string`